### DISCUSSION TITLE
Adding custom data under profile

### DISCUSSION THREAD

**Discussion by Nsbx** [bluesky-social/atproto#2942] [Bluesky Lexicons]
One of the things I liked about mastodon is the personalized information box.

I think this would be a very interesting addition, as my screenshot shows, it would allow you to add a link to a personal site, add the languages spoken, or even the artists who created your avatar or background.

But in itself, you can add whatever you like.

![image](https://github.com/user-attachments/assets/f75f65dd-a854-4f8c-a9d3-baf4f7b5e60c)

Edit : I found this proposal similar, but in my opinion it's “too closed”.
https://github.com/bluesky-social/atproto/discussions/2142

**Comment by bnewbold** [comment]
There are two aspects to this: one is the governance question of who gets to decide what info can be associated with an account in what context (what is the "profile"), especially across apps (eg, bsky distinct from smokesignals and whtwnd). The second is what the Bluesky schemas and apps will include.

For the first, the current recommendation is for apps and devs who are building clients to do this sort of extension with separate "sidecar" records. So create a `com.whtwnd.profile` record type with the additional fields, or `com.nsbx.profile`. Or get organized with other devs and do `org.atproto-collective.profile` and develop it as a group. Then get advocate for other app developers (like deck.blue, or the Bluesky app itself) to display that info. This is the vision for how this kind of extension would work in the atmosphere.

For the second, we are likely to add "account linking" and possibly some other fields to the Bluesky app itself. But i'm not sure we want to support an unlimited set of metadata fields: a lot of the Bluesky app experience is intentionally simple. This is an opinionated and subjective decision, and debatable (I assume many people would disagree!), but it isn't at the "protocol" level, it is at the application/client level.
